### DISCUSSION TITLE
Anyone tried linkedin Oauth?

### DISCUSSION THREAD

**Discussion by Oumar24sh** [feathersjs/feathers#2274] [Q&A]
### Steps to reproduce

(First please check that this issue is not already solved as [described
here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#report-a-bug))

- [x] Tell us what broke. The more detailed the better.
- [x] If you can, please create a simple example that reproduces the issue and link to a gist, jsbin, repo, etc. This makes it much easier for us to debug and issues that have a reproducable example will get higher priority.

### Expected behavior
Tell us what should happen

### Actual behavior
Tell us what happens instead

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

**NodeJS version**:

**Operating System**:

**Browser Version**:

**React Native Version**:

**Module Loader**:

**Comment by Oumar24sh** [comment]
i specificied the scope to be r_liteprofile like the documentation said.. but still got { strategy: 'linkedin',   error:    { oauth_problem:       'Scope "r_basicprofile" is not authorized for your application' } }  when going through the auth flow

**Comment by oceanchild** [comment]
same here
